components/dropdown/Dropdown.tsx

```diff
@@ -115,7 +121,10 @@ const Dropdown: React.FC<DropdownProps> = ({
                     ? styles.active
                     : styles.listOfCities
                 }
-                onClick={handleClick}
+                onClick={(e) => {
+                  e.preventDefault();
```

Comment:
I moved the e.preventDefault here to simplify the event handler function, that way you don't need to type the even being passed like `(e: ChangeEvent<HTMLInputElement>) => `